docs/gettinginvolved/cluster-setup.rst

```diff
@@ -185,20 +185,23 @@ The :program:`setup-cluster-containers` script has the following command line op
 
```

Comment:
We no longer have the --app-template parameter.
Now we have some more options. The mandatory ones are:
--image
-- mountpoint
The optional ones are:
--max-size
--wait-interval